v2.0/getting-started/docker/tutorials/advanced-policy.md

```diff
@@ -2,229 +2,292 @@
 title: Accessing Calico policy with Calico as a network plugin
 ---
 
-
 ## Background
 
-With Calico, a Docker network represents a logical set of rules that define the 
-allowed traffic in and out of containers assigned to that network.  The rules
-are encapsulated in a Calico "profile".
+With Calico as a Docker network plugin, Calico will create an identically named 
+[profile]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/profile)
+to represent each Docker network.  This profile is applied to each container 
```

Comment:
Something to think about: This section has turned into Calico-Docker 101. I wonder if we should flesh this out more with sample output and then move  it to  `getting-started/docker/index`. Then make this doc strongly recommend you read that other part first?